src/algebra/module.lean

```diff
@@ -512,9 +512,9 @@ by refine {smul := (•), ..};
 protected def subtype : p →ₗ[R] M :=
```

Comment:
The tool didn't rename the definition!

Reply:
Yup... a known bug (-;
I just pushed a manual fix.